anaconda-project.yml

```diff
@@ -0,0 +1,59 @@
+name: gapminder
+description: Gapminder test
+maintainers:
+- philippjfr
+
+channels:
+- pyviz
+- jbednar
+
+dependencies: &deps
+- nbserverproxy
+- altair
+- plotly
+- holoviews=1.12.2
+- hvplot=0.4.0
+- panel=0.5.1
+- pyct
+- nodejs
+- param=1.9.0
+- pyviz_comms=0.7.2
+- bokeh=1.1.0
+- tornado<6
+- notebook ==5.7.8
+
+packages: *deps
+
+platforms:
+- linux-64
+- osx-64
+- win-64
+
+commands:
+  dashboard:
+    unix: panel serve app.ipynb
+    supports_http_options: true
+  notebook:
+    notebook: app.ipynb
+    env_spec: pyviz-topics-gapminder
+  test:
+    unix: pytest --nbsmoke-run -k *.ipynb --ignore envs
+    windows: pytest --nbsmoke-run -k *.ipynb --ignore envs
+    env_spec: test
+  lint:
+    unix: pytest --nbsmoke-lint -k *.ipynb --ignore envs
+    windows: pytest --nbsmoke-lint -k *.ipynb --ignore envs
+    env_spec: test
+
+variables: {}
+
+env_specs:
+  pyviz-topics-gapminder:
+    channels: []
+    packages: []
+  test:
+    packages:
+    - nbsmoke ==0.2.8
+    - pytest ==4.4.1
+skip_imports:
+  environment: 2fba6dae4163fbed2318d5b0777e6d31eb8abc82
```

Comment:
what is this?

Reply:
Sigh. It is something anaconda-project insists on adding if you have an environment.yml...maybe it is a SHA of that file? I'll delete it.